backend/src/test/java/com/bakdata/conquery/integration/tests/TestUser.java

```diff
@@ -0,0 +1,26 @@
+package com.bakdata.conquery.integration.tests;
+
+import com.bakdata.conquery.models.auth.entities.User;
+import org.apache.shiro.authz.Permission;
+
+import java.util.Collection;
+import java.util.List;
+
+public class TestUser extends User {
+
+    public TestUser() {
+        super("user", "user");
+    }
+
+    public boolean isPermitted(Permission permission) {
+        return shiroUserAdapter.isPermitted(permission);
```

Comment:
könntest du den shiroUserAdapter in einem protected getter packen? das wäre glaube ich sauberer zu lesen.

Reply:
👍